examples/kubevirt-disk-uploader-tekton.yaml

```diff
@@ -212,17 +217,20 @@ metadata:
   name: kubevirt-disk-uploader-pipeline
 spec:
   params:
-  - name: POD_NAME
-    description: "Name of the virtual machine"
+  - name: EXPORT_SOURCE_KIND
+    description: "Kind of the export source"
+    type: string
+  - name: EXPORT_SOURCE_NAME
+    description: "Name of the export source"
     type: string
   - name: VOLUME_NAME
-    description: "Volume name of the virtual machine"
+    description: "Volume name of the export source"
```

Comment:
Probably worth to mention, that if user selects PersistentVolumeClaim, then the volumeName should be the same as source name

Reply:
I doubt if we should mention it, because VMExport API docs should mention it.

Reply:
I've mentioned it:
```
  - name: VOLUME_NAME
    description: "Volume name (If source kind is PVC, then volume name is equal to source name)"
```